### PR TITLE
Fix flakyness in TopDownBottomUpOfLoadedCapture.

### PR DESCRIPTION
**Pull request by danielfenner**
Merely adding an additional wait of 10s to give the system time to
react.

Bug: http://b/242843902
Test: local run - still works, the wait happens.

### CODE REVIEW

File: contrib/automation_tests/test_cases/connection_window.py [hunk 1]
```diff
@@ -58,6 +59,8 @@ def _execute(self, capture_file_path: str, expect_fail=False):
 
         others_button = self.find_control('Button', '...')
         others_button.click_input()
+        # There is a rare flakyness which causes the file dialog to open slowly: b/242843902
+        time.sleep(10)
```
Comment (karupayun) [inline contrib/automation_tests/test_cases/connection_window.py:63]: what about adding just 1 second? 10 seconds seems a lot.

Comment (danielfenner) [reply contrib/automation_tests/test_cases/connection_window.py:63]: From the video in the bug it looks as if we would need quite  a bit more time (we don't see the operation finish in ~20s)

Comment (danielfenner) [reply contrib/automation_tests/test_cases/connection_window.py]: Yeah, agree, that is way nicer.

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks

**Review by karupayun — APPROVED** [review]
LGTM! Thanks for addressing this issue and the comment! :)

**Pull request merged**